### DISCUSSION TITLE
Dropped Frames on YouTube HLS Streaming

### DISCUSSION THREAD

**Discussion by solidmgsnake** [obsproject/obs-studio#9486] [General]
When I attempt to stream to YouTube using the HLS (HTTP Live Streaming) protocol and the HEVC (High Efficiency Video Coding) codec, I encounter significant dropped frames when attempting to stream above 15,000 bitrate. This is puzzling to me since I've been able to successfully push much higher bitrates (e.g., 50,000 bitrate) using the RTMP protocol on YouTube without encountering dropped frames.

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
